drivers/volume/portworx/portworx.go

```diff
@@ -2855,7 +2854,13 @@ func (p *portworx) StartBackup(backup *storkapi.ApplicationBackup,
 			}
 			return true, nil
 		})
-		if err != nil || cloudBackupCreateErr != nil {
+		if err == nil {
+			// Only add volumeInfos if this was a successful backup
+			volumeInfos = append(volumeInfos, volumeInfo)
+		} else if err != nil || cloudBackupCreateErr != nil {
+			if _, ok := cloudBackupCreateErr.(*ost_errors.ErrCloudBackupServerBusy); ok {
+				return volumeInfos, &storkvolume.ErrStorageProviderBusy{Reason: cloudBackupCreateErr.Error()}
+			}
```

Comment:
We might need to append the volumeInfo to the list, when we hit ErrExists error.